### PR TITLE
Add arguments-per-line rule

### LINKED ISSUE #1000: One argument per continuation line
See style guide

### PR DESCRIPTION
**Pull request by bhirsz**
Closes #1000

### CODE REVIEW

File: docs/releasenotes/unreleased/rules.7.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+New rule arguments-per-continuation-line (#1000)
+------------------------------------------------
+
+Added new I0532 ``arguments-per-continuation-line`` rule.
+
+If the keywords ``[Arguments]`` are split into multiple lines, it is recommended to put only one argument per
```
Comment (mnojek) [inline docs/releasenotes/unreleased/rules.7.rst:6]:
```suggestion
If the keyword's ``[Arguments]`` are split into multiple lines, it is recommended to put only one argument per
```

File: docs/releasenotes/unreleased/rules.7.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+every line.
+
+Example of rule violation::
+
+    *** Keywords ***
+    Keyword With Multiple Arguments
+    [Arguments]    ${first_arg
```
Comment (mnojek) [inline docs/releasenotes/unreleased/rules.7.rst:13]:
```suggestion
    [Arguments]    ${first_arg}
```

File: docs/releasenotes/unreleased/rules.7.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+New rule arguments-per-continuation-line (#1000)
+------------------------------------------------
+
+Added new I0532 ``arguments-per-continuation-line`` rule.
```
Comment (mnojek) [inline docs/releasenotes/unreleased/rules.7.rst:4]: What about just `arguments-per-line`?

Comment (bhirsz) [reply docs/releasenotes/unreleased/rules.7.rst:4]: It does sound better. Also now I realized I'm only checking from the second line onwards. But I need to check it from the first line if the arguments settings is multiline. I will fix it

**Review by mnojek — APPROVED** [review]

File: robocop/checkers/lengths.py [hunk 4]
```diff
@@ -337,6 +338,41 @@
...
+        Bad |:x:| ::
+
+            *** Keywords ***
+            Keyword With Multiple Arguments
+            [Arguments]    ${first_arg
+            ...    ${second_arg}    ${third_arg}=default
+
+         Good |:white_check_mark:| ::
+
+        ..  code-block:: none
```
Comment (tnh8) [inline robocop/checkers/lengths.py:366]: This looks wrong in the documentation output: 

image

**Pull request merged**